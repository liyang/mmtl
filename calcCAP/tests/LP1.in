1
2
0
0
2
0
0
0.000700
0.000050
0.000347
0.000750
0.000002
0.00000055
0.000351
0.000750
0.000002
0.00000055
0
0.000050
0.000700
0.000700
12
0
0.000750
0.000700
0.000050
1
1024
40
6
8
10
5
0.0
1.0e-5
2
5
20000.0
