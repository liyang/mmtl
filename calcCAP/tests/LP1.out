---------------------------------------
Capacitance Values
Input file : /sppdg/software/src/tnt/calcCAP/tests/LP1.in
---------------------------------------
 [0][0] = 106.074
 [1][0] = -79.8756
 [0][1] = -79.8758
 [1][1] = 106.075

Estimated Maximum Computational Error is: 0

User CPU time: 19
 Sys CPU time: 0
